.circleci/config.yml

```diff
@@ -7,33 +7,23 @@ jobs:
     steps:
       - checkout
       - run: npm install
-      - run: npm run lint:features
+      - run: npm run lint
 
   test-with-python-hooks:
     docker:
-      - image: circleci/ruby:2.6-node
+      - image: circleci/python:3.7-node
     working_directory: ~/repo
     steps:
       - checkout
       - run:
           name: Install dev dependencies (Node.js)
           command: npm install
-      - run:
-          name: Install the BDD test suite dependencies (Ruby)
-          command: bundle install
       - run:
           name: Install the reference hooks implementation (Python)
-          command: |
-            curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
-            sudo python get-pip.py
-            sudo pip install dredd_hooks
+          command: sudo pip install dredd_hooks
```

Comment:
Not saying you have to change this, but you can avoid using sudo with:

```shell
$ pip install dredd_hooks --user
```

Then to use any binary it provides:

```shell
$ export PATH="$HOME/.local/bin:$PATH"
```

I believe Python would automatically pick up the library from `~/.local`

Reply:
That's neat. I think it doesn't matter in CI, but it's good to know this is so easy now. (Or was it the same in the past as well? 🤔 )